onnxruntime/__init__.py

```diff
@@ -26,4 +26,67 @@
 except ImportError:
     pass
 
+try:
+    from onnxruntime.training.ortmodule import ORTModule
+    has_ortmodule = True
+except: # noqa
+    has_ortmodule = False
+
+package_name = ''
+cuda_version = ''
+
+if has_ortmodule:
+    try:
+        # collect onnxruntime package name, version, and cuda version
+        from .build_and_package_info import package_name
+        from .build_and_package_info import __version__
+
+        cuda_version = None
+        try:
+            from .build_and_package_info import cuda_version
+        except: # noqa
+            pass
+
+        print('onnxruntime training package info: package_name:', package_name)
```

Comment:
Can you please use `stderr` for logs like this?

(Ideally we'd be able to plug in to some standard Python logging library, but I don't know if we want to take on the complexity just for printing these messages....)

Reply:
updated with file=sys.stderr at print calls